tests/test_complex_application.py

```diff
@@ -113,6 +113,7 @@ def test_complex():
     # |      |
     # |      |
     # t4     d4
+    os.environ["MODIN_PERSISTENT_PICKLE"] = "True"
```

Comment:
This should be set in conftest to be generalized. Same for unsettling the env variable.

Reply:
I think this should be set in the pickle file, since the main code will potentially run into trouble without it, but there's some uncertainties at the moment so I'm currently taking it slow in making such move

Reply:
Absolutely !